src/components/UniversalTable/UniversalTable.module.scss

```diff
@@ -3,7 +3,7 @@
 .table {
   &Container {
     width: 100%;
-    margin: 100px auto;
+    margin: 0 auto;
```

Comment:
why are yo so sure it won't brake anything on other pages that use the same component?

Reply:
On present time (i think so) we dosen`t have any pages, which uses this component.
This margin-top/bottom i was add to table during developing it.
Now, on Users page i`ve understood, that this component shouldnt have extra margins. 
Thats why it was deleted.

PS 100px margins dosen`t allow me put closer add button in this case, i`m sure in future we`ll faced with similar problems